src/components/notifications/notifications.tsx

```diff
@@ -47,6 +47,9 @@ export class Notifications extends React.Component<NotificationsProps> {
                         sel.removeAllRanges();
                         sel.addRange(range);
                     }
+                }} style={{
+                    overflowWrap: 'break-word',
```

Comment:
```suggestion
                    // fit long words by wrapping them instead of overflowing past the width
                    overflowWrap: 'break-word',
```

add the comment I mentioned [before](https://github.com/argoproj/argo-ui/pull/546#discussion_r1634122451)

Reply:
I unfortunately can't commit that myself via suggestion as a maintainer normally can, I'm guessing because of Fidelity's permissions

Reply:
Thanks, I will add the comment

Reply:
Hi @agilgur5 
Just added explanatory comment as suggested